label.py

```diff
@@ -72,7 +74,10 @@ def close(e):
 root.bind("<Escape>", close)
 root.mainloop()
 
+print(labels)
 
 # Store results.
-with open('labels.json', 'w') as f:
-    json.dump(labels, f)
+with open('labels.csv', 'w') as f:
+    writer = csv.writer(f, delimiter=',')
```

Comment:
Could you add a header?

Reply:
Yeah sure!

Reply:
@marco-c  please check! I have made the changes as asked.